validator/build.py

```diff
@@ -300,5 +320,7 @@ def RunValidatorTest(out_dir):
 GenerateValidateBin(out_dir='codegen')
 RunSmokeTest(out_dir='codegen')
 CompileValidatorTestMinified(out_dir='codegen')
-GenerateValidatorTest(out_dir='codegen')
-RunValidatorTest(out_dir='codegen')
+CompileHtmlparserTestMinified(out_dir='codegen')
```

Comment:
Codegen is somewhat an unusual name since we normally call these build or dist. Ok to leave for now. But please consider.